hack/update-fcos-bootimage.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+# Usage: ./hack/update-fcos-bootimage.py https://builds.coreos.fedoraproject.org/prod/streams/next/builds/32.20200517.1.0/x86_64/meta.json amd64
```

Comment:
Hopefully this will be obsolete with
https://github.com/openshift/installer/pull/4582

Reply:
Any objections to landing that one first?  It will simplify everything here I think because now FCOS/RHCOS will have the same public documented JSON format.  (Aside from some special casing like Azure)

Reply:
You could try rebasing this PR on that one?

I think `hack/update-fcos-bootimage.py` is then literally just e.g. `curl -L -o data/data/fcos.json https://builds.coreos.fedoraproject.org/streams/stable.json` or so.